### ISSUE TITLE
cannot load such file -- chef/version_class - chef dependency missing in gemspec

### ISSUE THREAD

**Issue by etagwerker** [chef/chef-zero#136]
Hi! 

I recently started using `chef-zero` and the first problem I ran into was this:

```
/Users/etagwerker/.rvm/gems/ruby-2.1.2@carbide/gems/chef-zero-4.2.2/lib/chef_zero/endpoints/policies_endpoint.rb:3:in `require': cannot load such file -- chef/version_class (LoadError)
	from /Users/etagwerker/.rvm/gems/ruby-2.1.2@carbide/gems/chef-zero-4.2.2/lib/chef_zero/endpoints/policies_endpoint.rb:3:in `<top (required)>'
	from /Users/etagwerker/.rvm/gems/ruby-2.1.2@carbide/gems/chef-zero-4.2.2/lib/chef_zero/server.rb:73:in `require'
	from /Users/etagwerker/.rvm/gems/ruby-2.1.2@carbide/gems/chef-zero-4.2.2/lib/chef_zero/server.rb:73:in `<top (required)>'
	from /Users/etagwerker/.rvm/gems/ruby-2.1.2@carbide/gems/chef-zero-4.2.2/bin/chef-zero:11:in `require'
	from /Users/etagwerker/.rvm/gems/ruby-2.1.2@carbide/gems/chef-zero-4.2.2/bin/chef-zero:11:in `<top (required)>'
	from /Users/etagwerker/.rvm/gems/ruby-2.1.2@carbide/bin/chef-zero:23:in `load'
	from /Users/etagwerker/.rvm/gems/ruby-2.1.2@carbide/bin/chef-zero:23:in `<main>'
	from /Users/etagwerker/.rvm/gems/ruby-2.1.2@carbide/bin/ruby_executable_hooks:15:in `eval'
	from /Users/etagwerker/.rvm/gems/ruby-2.1.2@carbide/bin/ruby_executable_hooks:15:in `<main>'
```

This happened because my project's `Gemfile` looked like this:

```
source 'https://rubygems.org'

group :development do
  gem 'chef-zero'
end
```

I assumed that `chef-zero` already had the `chef` dependency, but that's not the case. 

After reading PR #6, I realize that this is on purpose, but why? 

I've solved my problem by updating my `Gemfile`:

```
source 'https://rubygems.org'

group :development do
  gem 'chef'
  gem 'chef-zero'
end
```

I see two possible solutions: Adding the dependency, or updating the README (especially the [Installation](https://github.com/chef/chef-zero#installation) section)

Please let me know.

Thanks!

**Comment by fgimian** [comment]
This caught me out too, I think it should be included as a dependency to be honest.  The [description on ruby-gems](https://rubygems.org/gems/chef-zero/versions/4.2.3) says it's self-contained.

**Comment by pavannaganna** [comment]
i got this error too. where  can i find the Gemfile

**Comment by thommay** [comment]
This was fixed by #140 a fair while ago.
